### PR TITLE
refactor(meta): remove legacy frontend support in fragmenter

### PR DESCRIPTION
**Pull request by yezizp2012**
## What's changed and what's your intention?

As title.

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:

- Summarize your change (**mandatory**)
- How does this PR work? Need a brief introduction for the changed logic (optional)
- Describe clearly one logical change and avoid lazy messages (optional)
- Describe any limitations of the current code (optional)

## Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests

## Refer to a related PR or issue link (optional)
Resolve https://github.com/singularity-data/risingwave/issues/2557

### CODE REVIEW

**Comment by skyzh** [comment]
Would you please hold until https://github.com/singularity-data/risingwave/pull/2263 merges? It's a big refactor.

**Comment by yezizp2012** [comment]
> Would you please hold until #2263 merges? It's a big refactor.

Sure, no problem.

File: src/meta/src/dashboard/index.html [hunk 1]
```diff
@@ -108,7 +108,9 @@ <h1 class="flex-auto leading-6 font-semibold text-sky-500 dark:text-sky-400">
   </div>
 </body>
 
-<script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js"
```
Comment (yezizp2012) [inline src/meta/src/dashboard/index.html:111]: Replace `jquery` with a more stable src power by cloudflare.

**Review by fuyufjh — APPROVED** [review]
LGTM

**Pull request merged**